ZAPD/GameConfig.cpp

```diff
@@ -0,0 +1,138 @@
+#include "GameConfig.h"
+
+#include <functional>
+#include "Utils/Directory.h"
+#include "Utils/File.h"
+#include "Utils/Path.h"
+#include "tinyxml2.h"
+
+void GameConfig::ReadTexturePool(const std::string& texturePoolXmlPath)
+{
+	tinyxml2::XMLDocument doc;
+	tinyxml2::XMLError eResult = doc.LoadFile(texturePoolXmlPath.c_str());
+
+	if (eResult != tinyxml2::XML_SUCCESS)
+	{
+		fprintf(stderr, "Warning: Unable to read texture pool XML with error code %i\n", eResult);
+		return;
+	}
+
+	tinyxml2::XMLNode* root = doc.FirstChild();
+
+	if (root == nullptr)
+		return;
+
+	for (tinyxml2::XMLElement* child = root->FirstChildElement(); child != nullptr;
+	     child = child->NextSiblingElement())
+	{
+		if (std::string(child->Name()) == "Texture")
```

Comment:
Probably doesn't matter, but you can avoid a memory allocation + a string copy by using a string_view instead of a string. (string_view will still let you compare strings with ==) 

```suggestion
		if (std::string_view(child->Name()) == "Texture")
```

Reply:
Oh, this is neat. I think there's probably a lot of other places that could use this... well a lot that I didn't already turn into switches, anyway.